src/pages/logIn/LogInPage.tsx

```diff
@@ -1,33 +1,50 @@
-import { Box, Button, Main, MainHeadline, TextInput } from "../../ui";
-import React, { ChangeEvent, FormEvent, useState } from "react";
+import {
+  Box,
+  Button,
+  Main,
+  MainHeadline,
+  TextInput,
+  ValidationError,
+} from "../../ui";
+import React, { useState } from "react";
 import cx from "classnames";
 import { useAuthentication } from "../../features/authentication";
+import { Form, useValidation } from "usetheform";
+import {
+  validateEmail,
+  validateRequiredEmail,
+  validateRequiredPassword,
+} from "../utils/validation";
+
+interface FormState {
+  email: string;
+  password: string;
+}
+
+const initialFormState: FormState = {
+  email: "",
+  password: "",
+};
 
 export default function LogInPage() {
+  const [emailStatus, emailValidation] = useValidation([
+    validateRequiredEmail,
+    validateEmail,
+  ]);
+  const [passwordStatus, passwordValidation] = useValidation([
+    validateRequiredPassword,
+  ]);
   const { isLoggedIn } = useAuthentication();
-  const [email, setEmail] = useState("");
-  const [password, setPassword] = useState("");
   const [status, setStatus] = useState("new");
 
-  function handleChangeEmail(event: ChangeEvent<HTMLInputElement>) {
-    setEmail(event.target.value);
-  }
-
-  function handleChangePassword(event: ChangeEvent<HTMLInputElement>) {
-    setPassword(event.target.value);
-  }
-
-  async function handleSubmit(event: FormEvent<HTMLFormElement>) {
-    event.preventDefault();
+  async function handleSubmit({ email, password }: FormState) {
```

Comment:
The Form onSubmit accepts either Promises or sync function. 
To proper use it you should change a bit the implementation like : 

```
  async function handleSubmit({ email, password }: FormState) {
    try {
      await window.firebase.auth().signInWithEmailAndPassword(email, password);
      setStatus("success");
    } catch (error) {
      switch (error.code) {
        case "auth/user-not-found":
        case "auth/wrong-password":
          setStatus("error/bad-credentials");
          break;
        default:
          setStatus("error/unknown-cause");
      }
      throw error;
    }
 
  }
```
Please have a look at:  https://iusehooks.github.io/usetheform/docs-form

Your implementation is working but is not well counting the submitted (success) and submitAttempts values beacuse as it is now it is always resolved with success.

```
const { submitted, submitAttempts } = useForm();
```